api/pkg/transformer/server/grpc/server.go

```diff
@@ -106,7 +107,8 @@ func (us *UPIServer) Run() {
 	}
 
 	m := cmux.New(lis)
-	grpcLis := m.Match(cmux.HTTP2HeaderField("content-type", "application/grpc"))
+	// cmux.HTTP2MatchHeaderFieldSendSettings ensures we can handle any gRPC client.
```

Comment:
What issue did you encounter when using `HTTP2HeaderField` ?

Reply:
the request from `https://github.com/grpc-ecosystem/grpc-health-probe/` couldn't redirected to current grpc, and getting timeout hence the liveness probe is failing